source/docs/software/commandbased/what-is-command-based.rst

```diff
@@ -54,59 +51,25 @@ Subsystems and commands
-currently-scheduled commands, and removing commands that have finished
-or been interrupted. The scheduler’s ``run()`` method may be called from
-any place in the user’s code; it is generally recommended to call it
-from the ``robotPeriodic()`` method of the ``Robot`` class, which is run
-at a default frequency of 50Hz (once every 20ms).
-
-Multiple commands can run concurrently, as long as they do not require
-the same resources on the robot. Resource management is handled on a
-per-subsystem basis: commands may specify which subsystems they interact
-with, and the scheduler will never schedule more than one command
-requiring a given subsystem at a time. this ensures that, for example,
-users will not end up with two different pieces of code attempting to
-set the same motor controller to different output values. If a new
-command is scheduled that requires a subsystem that is already in use,
-it will either interrupt the currently-running command that requires
-that subsystem (if the command has been scheduled as interruptible), or
-else it will not be scheduled.
-
-Subsystems also can be associated with “default commands” that will be
-automatically scheduled when no other command is currently using the
-subsystem. This is useful for continuous “background” actions such as
-controlling the robot drive, or keeping an arm held at a setpoint.
+Commands are run by the ``CommandScheduler``, a singleton class that is at the core of the command-based library. The ``CommandScheduler`` is in
+charge of polling buttons for new commands to schedule, checking the resources required by those commands to avoid conflicts, executing currently-scheduled commands, and removing commands that have finished or been interrupted. The scheduler’s ``run()`` method may be called from any place in the user’s code; it is generally recommended to call it from the ``robotPeriodic()`` method of the ``Robot`` class, which is run at a default frequency of 50Hz (once every 20ms).
+
+Multiple commands can run concurrently, as long as they do not require the same resources on the robot. Resource management is handled on a
+per-subsystem basis: commands may specify which subsystems they interact with, and the scheduler will never schedule more than one command requiring a given subsystem at a time. this ensures that, for example, users will not end up with two different pieces of code attempting to set the same motor controller to different output values. If a new command is scheduled that requires a subsystem that is already in use,
+it will either interrupt the currently-running command that requires that subsystem (if the command has been scheduled as interruptible), or else it will not be scheduled.
+
+Subsystems also can be associated with “default commands” that will be automatically scheduled when no other command is currently using the subsystem. This is useful for continuous “background” actions such as controlling the robot drive, or keeping an arm held at a setpoint.
```

Comment:
This could be interpreted that your robot drive will stop working when another command is using the subsystem, when this is incorrect. Can this be worded differently?

Reply:
Which part in particular can be read that way?  I'm not seeing it.

Reply:
I think @Daltz333 was referring to:
>Subsystems also can be associated with “default commands” that will be automatically scheduled when no other command is currently using the subsystem. This is useful for continuous “background” actions such as controlling the robot drive, or keeping an arm held at a setpoint.

But I don't see anything wrong with this statement.